packages/generator/ni_measurement_plugin_sdk_generator/client/_support.py

```diff
@@ -356,6 +353,12 @@ def validate_measurement_service_classes(measurement_service_classes: List[str])
         raise click.ClickException("No registered measurements.")
 
 
+def replace_enum_class_type(output: str) -> str:
+    """Formats the data with enum type."""
```

Comment:
This docstring is vague and having an input named "output" doesn't help.

Reply:
I have updated the the input parameter name and docstring. Also, I have moved this method to `mako` as per the suggestion from other [comment](https://github.com/ni/measurement-plugin-python/pull/901#discussion_r1767497806)